src/DependencySupport.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Class to determine support for AMP plugin features.
+ *
+ * @since 2.1.2
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP;
+
+use AmpProject\AmpWP\Infrastructure\Delayed;
+use AmpProject\AmpWP\Infrastructure\Service;
+
+/**
+ * DependencySupport class.
+ *
+ * @internal
+ */
+class DependencySupport implements Service, Delayed {
+
+	/**
+	 * The minimum version of Gutenberg supported.
+	 *
+	 * @var string
+	 */
+	const GB_MIN_VERSION = '9.2.0';
+
+	/**
+	 * The minimum version of WordPress supported.
+	 *
+	 * @var string
+	 */
+	const WP_MIN_VERSION = '5.6';
+
+	/**
+	 * Get the action to use for registering the service.
+	 *
+	 * @return string Registration action to use.
+	 */
+	public static function get_registration_action() {
+		return 'plugins_loaded';
+	}
```

Comment:
Is `Delayed` necessary since the services get registered at `plugins_loaded` anyway?

Reply:
Resolving since outdated since https://github.com/ampproject/amp-wp/pull/6225/commits/260734553a6bbe667fa5d05961c314cc99c37fe3.